src/main/config/config.c

```diff
@@ -681,7 +681,7 @@ void validateAndFixGyroConfig(void)
                 motorConfigMutable()->dev.motorPwmProtocol = PWM_TYPE_DSHOT300;
             }
             if (gyro.sampleRateHz > 4000) {
```

Comment:
:thinking: Seems this will be 8000 for ICM/MPU gyro's - otherwise we would be able to set a higher pid_denom before forcing it to 2.